src/AvtaleSide/AvtaleStatus/AvtaleStatus.less

```diff
@@ -65,11 +67,12 @@
 
         &__infotekst {
             border-bottom: 0;
+            border-bottom: 1px solid #c6c2bf;
         }
 
         &__header {
             margin-top: 0rem;
-            margin-bottom: 1rem;
+            margin-bottom: 1rem !important;
```

Comment:
Grunnen til at du har måttet bruke `!important` for å få css-regelen din til å virke her (og sikkert flere andre steder) er at `H1` elementet har fått satt en margin-regel et annet sted med en mer spesifikk selector. Spesifikt så har `<Innholdstittel>` fra designsystemet (som egentlig produserer en `H1`) en mer spesifikk margin-regel.

Det vil nok antagelig være mer fornuftig å wrappe disse typografi-elementene fra designsystemet i `div`-er el. og så sette plassering på disse isteden. 

Har laget en liten gif som forklarer hva jeg mener 🙂
![css](https://user-images.githubusercontent.com/5412607/72143683-71c47480-3397-11ea-8b3a-d8e50f10f396.gif)
(`.typo-innholdstittel` er css-klassen `<Innholdstittel>` får med seg fra designsystemet)